onbuild/Dockerfile

```diff
@@ -1,22 +1,25 @@
 FROM ubuntu:16.04
 MAINTAINER Event Store LLP <ops@geteventstore.com>
 
-ENV ES_VERSION=4.0.1
+ENV ES_VERSION=4.0.1 \
+    DEBIAN_FRONTEND=noninteractive \
+    EVENTSTORE_CLUSTER_GOSSIP_PORT=2112
 
 RUN apt-get update \
-    apt-get install tzdata \
-    && apt-get install curl -y \
+    && apt-get install tzdata curl iproute2 -y \
     && curl -s https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/script.deb.sh | bash \
     && apt-get install eventstore-oss=$ES_VERSION -y \
+    && apt-get autoremove \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-EXPOSE 1113
-EXPOSE 2113
+EXPOSE 112 2112 1113 2113
```

Comment:
This appears to be a typo, is this `112` not supposed to be `1112` (The internal tcp port)?